### PR TITLE
[Feat]: Issue #14 - Add Verdana Base Font Family

### LINKED ISSUE #14: Facilitate Sans Serif font usage for general UI elements
Add pluggable themes for folks who have a preference to Sans Serif fonts

### PR DESCRIPTION
**Pull request by mcastre**
## Link to the Issue

Resolves [Issue #14 ](https://github.com/s-katte/Sketchify/issues/14)

## Description of your PR

Set the base font family of the application to Verdana in index.css.

## Screenshots (if applicable)

image

## Test Plan

Can delete line 6 in index.css to see app revert to the original font for comparison.

### Have you read the [Contributing Guidelines](https://github.com/s-katte/React-Code-Pen/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -3,6 +3,7 @@ body,
 #root {
     margin: 0;
     height: 100%;
+    font-family: 'Verdana', sans-serif;
```
Comment (gridhead) [inline src/index.css]: Please use Inter (https://fonts.google.com/specimen/Inter) and not Verdana

**Review by gridhead — CHANGES_REQUESTED** [review]
I have requested some changes. Please follow through.

**Review by s-katte — CHANGES_REQUESTED** [review]
@mcastre, please do the changes suggested by @t0xic0der.

**Comment by mcastre** [comment]
@t0xic0der @s-katte PR has been updated with requested changes

**Comment by mcastre** [comment]
Sorry, forgot to update from master. Excuse the extra commits.

**Review by s-katte — APPROVED** [review]
Looks fine to me.
Merging it.

**Pull request merged**